frontend/.eslintrc.json

```diff
@@ -29,69 +28,20 @@
         "never"
       ],
       "mocha/no-exclusive-tests": "error",
-      "no-var": "error",
-      "prefer-arrow-callback": "warn",
-      "no-unused-vars": "warn",
-      "no-warning-comments": "warn",
-      "no-console": 0,
       "indent": [
```

Comment:
why not keeping the indent rule in the root?

Reply:
it seems like those are different rules - you can't use the eslint rule for typescript. I actually did put all in root but then I noticed that it didn't report me indent for class properties/methods correctly. Therefore for each eslint basic rule I decided to prefer a same named rule in typescript plugin if it exists.